src/client/components/EditUserForm/CountryRoles/CountryRoles.tsx

```diff
@@ -28,7 +28,7 @@ type ModalOptionsProps = {
 }
 
 type Props = {
-  onChange: (value: Array<Partial<UserRole<RoleName>>>, key: string) => void
+  onChange: (value: Array<Partial<UserRole<RoleName>>>) => void
```

Comment:
you dont need to pass onChange here. this component can dispatch the action directly